incubator/gogs/README.md

```diff
@@ -0,0 +1,70 @@
+# Gogs Helm Chart
+
+[Gogs][] is a painless self-hosted Git service.
+
+## TL;DR;
+
+```console
+$ helm install incubator/gogs
+```
+
+## Introduction
+
+This chart bootstraps a [Gogs][] deployment on a [Kubernetes][] cluster using
+the [Helm][] package manager.
+
+## Prerequisites Details
+
+* PV support on underlying infrastructure (if persistence is required)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release incubator/gogs
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes nearly all the Kubernetes components associated with the
+chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Gogs chart and
+their default values.
+
+| Parameter                        | Description                                                  | Default                                                    |
+| -----------------------          | ----------------------------------                           | ---------------------------------------------------------- |
+| `image.repository`               | Gogs image                                                   | `gogs/gogs`                                                |
+| `image.tag`                      | Gogs image version                                           | `0.9.141`                                                  |
```

Comment:
I'm inclined to leave it as the 0.9.141 as opposed to specifying an RC, and it's an easy thing to override for an installation.